requirements/constraints.txt

```diff
@@ -8,9 +8,5 @@
 # pin when possible.  Writing an issue against the offending project and
 # linking to it here is good.
 
-# This packages is a backport which can only be installed on Python 2.7
-futures ; python_version == "2.7"
-
-# A dependency of pytest.  Pytest doesn't constrain it and 6.0.0 onward
-# only works with python 3
-more-itertools<6.0.0
+# Constraint from astroid 2.3.3
```

Comment:
why adding constraint here ?

Reply:
Make upgrade fails if that constraint is not present, I took that constraint fro  edx-platform